src/tools/cursors/MouseCursor.js

```diff
@@ -0,0 +1,66 @@
+import toolColors from '../../stateManagement/toolColors.js';
+
+const mousePosition = '8 8';
```

Comment:
Can we split these into x/y?

Reply:
Yeah I guess we can and then we can construct it as needed when we generate the cursor.

Reply:
Done.